doc/start/hardware-recommendations.rst

```diff
@@ -77,7 +77,8 @@ the OSD attempts to consume with the ``osd_memory_target`` configuration
 option.
 
 - Setting the osd_memory_target below 2GB is typically not recommended (it may
```

Comment:
:confval:

Reply:
Also: "mds_cache_memory" should be "mds_cache_memory_limit", as far as I can tell. I've made that alteration, too.